src/main/java/dev/jbang/cli/Alias.java

```diff
@@ -96,7 +96,7 @@ public Integer doCall() {
 		RunContext ctx = getRunContext();
 		Code code = ctx.forResource(scriptMixin.scriptOrFile);
 		if (name == null) {
-			name = CatalogUtil.nameFromRef(ctx.getOriginalRef());
+			name = CatalogUtil.nameFromRef(scriptMixin.scriptOrFile);
```

Comment:
is that an intentional change?

Reply:
Yes, it just doesn't make sense to put `scriptOrFile` into the `RunContext` to use it again a couple of lines down and nowhere else.